roles/security_errata/tasks/CVE-2015-0235.yml

```diff
@@ -0,0 +1,40 @@
+- name: copy ghost detection tool
+  copy:  src=detect_CVE-2015-0235 dest=/usr/local/bin/detect_CVE-2015-0235 mode=0755
+
+- name: get md5 of ghost detection tool
+  stat: path=/usr/local/bin/detect_CVE-2015-0235
+  register: st
+
+- name: error if md5 of ghost detection tool is not what we expect
+  fail: msg="checksum of /usr/local/bin/detect_CVE-2015-0235 failed.  Check for tampering."
+  when: st.stat.md5 != 'cc4ef149ea245f88825b24db141ddcf1'
+
+- name: run ghost detection tool
+  command: /usr/local/bin/detect_CVE-2015-0235
+  register: ghost
+
+- name: patch CVE-2015-0235
+  apt: pkg=libc6 state=latest
+  when: ghost.stdout == 'vulnerable'
+
+- name: list of processes using libc
+  shell: /usr/bin/lsof | grep libc | awk '{print $1}' | sort | uniq | xargs | logger -i -t CVE-2015-0235
+  register: libc
+  when: ghost.stdout == 'vulnerable'
```

Comment:
yeah, it's logging the results to syslog.   I used register it in case we want to turn it into a debug message or something else.